NSSCodeForNashville/NashvilleHistory
# Ddw bookmarks

## Status
**READY/IN DEVELOPMENT**

## Documentation

[ X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description

Markers can now be added and deleted from a user's bookmarks. 

## Resolves Issue Number

#27 #25 

## Deploy Notes
Notes regarding deployment the contained body of work. 

```sh
git pull --prune
git checkout ddw-bookmarks
ionic serve --lab
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Login as a user
1. Save a marker to your bookmarks
1. Make sure that marker shows up in the bookmarks page
1. Remove a bookmark
1. Make sure a marker cannot be added twice to your bookmarks

## Impacted Files in Application
List general components of the application that this PR will affect:

* BookmarkCtrl.js
* bookmarks.html
* MarkerCardsCtrl.js
* markers.html
* AllPlacesCtrl.js
* all-places.html
